centreon/tests/e2e/features/Cloud-notifications/01-notification-create/index.ts

```diff
@@ -293,21 +309,179 @@ Then(
   (contactSettings) => {
     switch (contactSettings) {
       case 'a single contact':
-        notificationSentCheck({ log: `<<${data.hosts.host2.name}>>` });
+        if (globalResourceType) {
+          notificationSentCheck({ logs: `<<${data.hosts.host2.name}>>` });
+        } else {
+          const logsToCheck = Array.from(
+            { length: 1000 },
+            (_, i) => `<<${data.hosts.host1.name}/service_${i + 1}`
```

Comment:
is it `i + 1` to have the services numbered from 1 to 1000 instead of 0 to 999? does it serve any practical purpose?

Reply:
@paul-oureib yes from 1 to 1000, the services are named from `service_1 ` to `service_1000`